src/smallstack.rs

```diff
@@ -113,6 +104,24 @@ impl<'a> VM<'a> {
             None => panic!("VM error: no argument supplied to `typeof`"),
         },
 
+        &["call", "primitive", "convert"] => if let
```

Comment:
Necessary to allow this: https://github.com/a3gis/smallstack-js/blob/master/examples/prime.js#L28